src/Components/ReviewPage.jsx

```diff
@@ -1,14 +1,25 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
-import { getReviewById } from "../utils/api";
+import { getReviewById, PatchVoteOnReview } from "../utils/api";
 import ErrorPage from "./ErrorPage";
 import "../css/ReviewPage.css";
 
 const ReviewPage = () => {
+  const [newVote, setNewVote] = useState(0);
   const [error, setError] = useState(false);
   const [review, setReview] = useState([]);
   const { review_id } = useParams();
 
+  const addVote = (VoteValue) => {
+    setNewVote((currVoteValue) => (currVoteValue += VoteValue));
+    PatchVoteOnReview(review.review_id, VoteValue)
+      .then(() => {})
+      .catch((err) => {
+        setNewVote((currVoteValue) => (currVoteValue -= VoteValue));
+        setError(true);
```

Comment:
when there is an error in voting the whole page disappears!

did you want this to happen?